src/Entity/MessageTemplate.php

```diff
@@ -282,7 +305,9 @@ public static function create(array $values = []) {
    * {@inheritdoc}
    */
   public function preSave(EntityStorageInterface $storage) {
-    $this->text = array_filter($this->text);
+    $this->text = array_filter($this->text, function ($a) {
```

Comment:
I've fixed this (not yet pushed). I think `partial` makes more sense than `delta`, which is fairly generic.